packages/website/src/bin/page-generator/index.js

```diff
@@ -377,6 +385,9 @@ module.exports = async function generatePages(
   if (typeof options.showExamples !== 'undefined') {
     pkgOpts.showExamples = options.showExamples;
   }
+  if (typeof options.customPackageFields !== 'undefined') {
```

Comment:
isn't that the same as `options.customPackageFields === undefined`?

Reply:
Nice, I've cleaned these up